patchkit-launcher-qt-src/src/lockfile.cpp

```diff
@@ -4,6 +4,13 @@
 #include "locations.h"
 #include "customexceptions.h"
 
+#if defined (Q_OS_UNIX)
```

Comment:
1. Does this also include Mac OSX?
2. Windows should be also supporting flock()
3. Maybe this http://doc.qt.io/qt-5/qlockfile.html will be more usable than using flock() directly?